src/socket/addrinfo.cr

```diff
@@ -189,6 +189,8 @@ class Socket
         addrinfo.value.ai_addr.as(LibC::SockaddrIn6*).copy_to(pointerof(@addr).as(LibC::SockaddrIn6*), 1)
       when Family::INET
         addrinfo.value.ai_addr.as(LibC::SockaddrIn*).copy_to(pointerof(@addr).as(LibC::SockaddrIn*), 1)
+      else
+        # TODO: (asterite) UNSPEC and UNIX unsupported?
```

Comment:
ditto `(asterite)` mention

Reply:
Same, I don't know why the other cases were left out and where leaving it like this is good or not.